app/src/androidTest/java/com/android/unio/components/BottomNavigationTest.kt

```diff
@@ -47,7 +50,11 @@ class BottomNavigationTest : TearDown() {
     MockKAnnotations.init(this)
     eventRepository = mock { EventRepository::class.java }
     eventViewModel =
-        EventViewModel(eventRepository, imageRepository, associationRepositoryFirestore)
+        EventViewModel(
+            eventRepository,
+            imageRepository,
+            associationRepositoryFirestore,
+            eventUserPictureRepositoryFirestore)
```

Comment:
When mocking dependencies in tests, it's generally better to mock an interface rather than a concrete implementation. You can mock an `eventUserPictureRepository`, it works the same for me when running the test.

Reply:
Not sure what you mean by this, he did mock the repository

Reply:
It means he mocks an `eventUserPictureRepositoryFirestore` not a `eventUserPictureRepository`
I would have explained more but he told me it was a draft PR and I should just just give him UI feedback

Reply:
The other repositories used in the test class, like `associationRepositoryFirestore` are also the Firestore ones

Reply:
> It means he mocks an `eventUserPictureRepositoryFirestore` not a `eventUserPictureRepository` I would have explained more but he told me it was a draft PR and I should just just give him UI feedback

Ah right I didn't read it properly